rhc-ose-ansible/inventory/ose-provision

```diff
@@ -12,6 +12,33 @@ openstack_key_name=""
 #openshift_openstack_master_storage_size: 10
 #openshift_openstack_node_storage_size: 10
 
+#env_id=
+rhc_ose_inv_dest=.
+rhc_ose_cluster_method=native
+rhc_ose_cluster_public_hostname=public.hostname.com
+rhc_ose_cluster_hostname=hostname.com
+#rhc_ose_ssh_user=root
+#rhc_ose_identity_provider=htpasswd
+deployment_type=openshift-enterprise
+
+#Openshift Components
+rhc_ose_enable_efk=true                  #If E-F-K logging stack should be deployed
```

Comment:
I had set this up for some consistency. for example if just using their vars we would have:
```...
rhc_ose_enable_efk=true (my efk doesnt have a `deploy=true` variable equivalent just a cleanup variable)
openshift_hosted_metrics_deploy=true
...```
 Just didnt think it looks very nice and also a bit misleading. But that was the reasoning for that. I could change it back which wouldn't be a problem.

Also i am not using that metrics playbook but the openshift_metrics role: https://github.com/openshift/openshift-ansible/tree/master/roles/openshift_metrics.

Maybe they have some openshift_ansible has some redundancies? I think we are better off sticking to the roles as opposed to that adhoc playbook. Let me know what you think.